src/api/spec/controllers/trigger_workflow_controller_spec.rb

```diff
@@ -1,38 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe TriggerWorkflowController, type: :controller, beta: true do
-  let(:user) { create(:confirmed_user, :in_beta, login: 'foo') }
```

Comment:
Local variables are defined in the context where they are needed to clarify what is needed in a specific context. It's much easier to fix specs with this approach.